lib/helpers/filter$lookup.js

```diff
@@ -0,0 +1,66 @@
+import {
+  isPlainObject,
+  reject,
+  isArray,
+  get,
+  includes,
+  has,
+  map,
+  omitBy,
+  mapValues
+} from 'lodash';
+
+const ALLOWED_LOOKUPS = [
+  'personas',
+  'personaIdentifiers',
+  'personaAttributes',
+  'journeys',
+  'journeyprogresses',
+  'statements',
+];
+
+const shouldRemove = (value, key) => {
```

Comment:
Since this looks similar to [remove$out](https://github.com/LearningLocker/learninglocker/blob/master/lib/helpers/remove$out.js), could this be simplified to the code below?

```js
const shouldRemove = (value, key) => {
  return (
    _.has(value, ['$lookup', 'from') &&
    includes(ALLOWED_LOOKUPS, get(value, ['$lookup', 'from']))
  );
}
```